### ISSUE TITLE
Brew cask support

### ISSUE THREAD

**Issue by TribuneX** [zkokaja/Brewlet#19]
I am installing most of my apps via [homebrew cask](https://formulae.brew.sh/cask/). It would be nice if brewlet could also notify me for updates for my casks.

**Comment by zkokaja** [comment]
I completely agree — it's a feature I've had in my backlog for quite a while. I think brew recently changed the API so it might be even easier now than before. Keep an eye out for the next release!

**Comment by zkokaja** [comment]
I created a pull request to `brew` to add data to their json API that will help with this feature: https://github.com/Homebrew/brew/pull/10278

**Comment by pvegh** [comment]
I'd like this. I always find my casks outdated

**Comment by zkokaja** [comment]
Commit 6027ad4 adds support for casks by using v2 of the JSON API brew provides. This means that outdated casks will now appear in the Packages menu list. I currently treat casks as formulae since they share corer common attributes, but am open to feedback.

Note that brew does not upgrade all casks, see this FAQ: https://github.com/Homebrew/brew/blob/120dab281625f4d2901f87cfa9597b881251a688/docs/FAQ.md#why-arent-some-apps-included-during-brew-upgrade. I don't see the need currently to implement a `--greedy` option.
